src/js/themes/media-theme-demuxed.js

```diff
@@ -0,0 +1,296 @@
+/* 
+<media-theme-demuxed>
+  <video
+    slot="media"
+    src="https://stream.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/high.mp4"
+  ></video>
+</media-theme-demuxed>
+*/
+
+import { defineCustomElement } from '../utils/defineCustomElement.js';
+import MediaTheme from './media-theme.js';
+
+const template = `
+<style>
+  :host {
+    --primary-color: black;
+    --secondary-color: transparent;
+    --tertiary-color: #7596CC;
+  }
+
+  video {
+    object-fit: initial;
+  }
+
+  media-controller {
+    width: 100%;
+    height: 100%;
+
+    --media-control-background: var(--secondary-color);
+    --media-control-hover-background: transparent;
+    
+    --media-icon-color: var(--primary-color);
+
+    --media-range-track-height: 6px;
+    --media-range-track-background: var(--tertiary-color);
+    --media-range-track-border-radius: 9999px;
+
+    --media-range-thumb-background: var(--tertiary-color);
+    --media-range-thumb-width: 14px;
+    --media-range-thumb-height: 14px;
+  }
+
+  media-control-bar {
+    position: relative;
+    margin: 30px;
+    padding: 10px 14px;
+    border-radius: 9999px;
+    background: rgba(0,0,0,0.2);
+  }
+
+  media-control-bar :first-child {
+    margin: 0 5px 0 0;
+  }
+  
+  media-control-bar :last-child {
+    margin: 0 0 0 5px;
+  }
+
+  media-play-button, media-mute-button, media-seek-forward-button, media-captions-button, media-pip-button, media-fullscreen-button, media-settings-button, media-cinema-button {
```

Comment:
In case it's helpful, since this is part of a theme you could add a class to all of these elements to make this simpler.

Reply:
Thanks for this suggestion, makes total sense and cleans things up nicely.